Dockerfile

```diff
@@ -1,23 +1,48 @@
 FROM python:3.12-slim AS base
 
+ARG UID=1000
+ARG GID=1000
+
 RUN apt-get update && apt-get install -y \
-    build-essential \
-    libpq-dev \
-    ffmpeg \
-    && rm -rf /var/lib/apt/lists/*
+      ffmpeg \
```

Comment:
We need the `ffprobe` utility from `ffmpeg`. I believe we can use `--no-install-recommends ffmpeg `to avoid installing any optional dependencies.

Do you want me to test this branch locally with `--no-install-recommends ffmpeg`?

Reply:
I have updated the PR. You can test it now. Let me know if something doesn't work.

Reply:
I tested this branch locally and have added the issue related to entrypoint.sh in another comment.